client/client.go

```diff
@@ -51,14 +97,14 @@ func New(apiEndpoint string) *KalaClient {
 //		id, err := c.CreateJob(body)
 func (kc *KalaClient) CreateJob(body *job.Job) (string, error) {
 	id := &api.AddJobResponse{}
-	resp, err := kc.requester.New().Post(api.JobPath).BodyJSON(body).ReceiveSuccess(id)
+	_, err := kc.do(http.MethodPost, kc.url(api.JobPath), http.StatusCreated, body, id)
```

Comment:
`resp` should be returned so that `defer resp.Body.Close()` can be run, or else we're leaking file descriptors